lib/please_cli/please_cli/config.py

```diff
@@ -433,24 +433,24 @@
         ],
         'deploy': 'HEROKU',
         'deploy_options': {
-            #'testing': {
-            #    'heroku_app': 'shipit-testing-codecoverage',
-            #    'heroku_dyno_type': 'web',
-            #    'url': 'https://codecoverage.shipit.testing.mozilla-releng.net',
-            #    'dns': 'codecoverage.shipit.testing.mozilla-releng.net.herokudns.com',
-            #},
-            #'staging': {
-            #    'heroku_app': 'shipit-staging-codecoverage',
-            #    'heroku_dyno_type': 'web',
-            #    'url': 'https://codecoverage.shipit.staging.mozilla-releng.net',
-            #    'dns': 'codecoverage.shipit.staging.mozilla-releng.net.herokudns.com',
-            #},
-            #'production': {
-            #    'heroku_app': 'shipit-production-codecoverage',
-            #    'heroku_dyno_type': 'web',
-            #    'url': 'https://codecoverage.shipit.mozilla-releng.net',
-            #    'dns': 'codecoverage.shipit.mozilla-releng.net.herokudns.com',
-            #},
+            'testing': {
+                'heroku_app': 'shipit-testing-codecoverage',
+                'heroku_dyno_type': 'web',
+                'url': 'https://codecoverage.testing.mozilla-releng.net',
+                'dns': 'codecoverage.testing.mozilla-releng.net.herokudns.com',
+            },
+            'staging': {
+                'heroku_app': 'shipit-staging-codecoverage',
+                'heroku_dyno_type': 'web',
+                'url': 'https://codecoverage.staging.mozilla-releng.net',
+                'dns': 'codecoverage.staging.mozilla-releng.net.herokudns.com',
+            },
+            'production': {
+                'heroku_app': 'shipit-production-codecoverage',
+                'heroku_dyno_type': 'web',
+                'url': 'https://codecoverage.mozilla-releng.net',
```

Comment:
Let's use `codecoverage.moz.tools` or `coverage.moz.tools` (and `codecoverage-ui.moz.tools` or `coverage-ui.moz.tools` for the frontend).
Otherwise, `codecoverage-backend.moz.tools` or `coverage-backend.moz.tools` (and `codecoverage.moz.tools` or `coverage.moz.tools` for the frontend).

I would probably prefer the first, but I have no strong opinions.